app/lib/submission_builder/ty2022/states/az/documents/az322.rb

```diff
@@ -17,7 +17,9 @@ def document
                     xml.Contributions contribution.amount.round
                   end
                 end
-                xml.TotalContributionsContSheet calculated_fields.fetch(:AZ322_LINE_4)
+                if calculated_fields.fetch(:AZ322_LINE_4) > 0
```

Comment:
Tim added a convenience method called `add_non_zero_claimed_value` that we should be using for cases like this